Source/Fuse.Controls.Panels/WrapPanel.uno

```diff
@@ -82,12 +82,21 @@ namespace Fuse.Controls
 			set { _wrapLayout.FlowDirection = value; }
 		}
 
-		/** The alignment of the content within the wrap panel rows. */
+		/**
+			@deprecated 2017-07-07
+		*/
 		public Alignment RowAlignment
 		{
 			get { return _wrapLayout.RowAlignment; }
 			set { _wrapLayout.RowAlignment = value; }
```

Comment:
Is this not enough? 

https://github.com/bolav/fuselibs-public/blob/WrapPanel_ContentAlignement/Source/Fuse.Controls.Panels/Layouts/WrapLayout.uno#L77